examples/agent/server.py

```diff
@@ -35,7 +36,19 @@ def get_eugene_thoughts(query: str) -> list:
     ]
 )
 
-llm = ChatOpenAI()
+
+llm = ChatOpenAI(temperature=0.5).configurable_alternatives(
```

Comment:
These options do not show up in the agent executor (but they do show up for the llms endpoint)